src/test_doubly.py

```diff
@@ -0,0 +1,96 @@
+# _*_ encoding: utf-8 _*_
+import pytest
+"""Test linked_list.py."""
+
+INSERT_ITEMS = [(['one', 'two', 'three', 'four'], 4)]
+PREV_VAL = [(['one', 'two', 'three', 'four'], 'two')]
+POP = [(['one', 'two', 'three', 'four'])]
+APPEND = [(['one', 'two', 'three'], 'three')]
+SHIFT = [(['one', 'two', 'three', 'four'], 'four'),
+         (['five', 'hat', 'bunny', 'dude'], 'dude')]
+EMPTY = [([])]
+REMOVE = [(['one', 'two', 'three', 'four', 'five'], 'three', 'two', 'four', 4),
+          (['Hello', 'This', 'is', 'a', 'test'], 'a', 'is', 'test', 4)]
+
+
+def test_inheritance():
+    from doubly_linked import DoublyLinked
+    from linked_list import LinkedList
+    new_list = DoublyLinked()
+    assert isinstance(new_list, LinkedList)
+
+
+@pytest.mark.parametrize('li, result', INSERT_ITEMS)
+def test_insert(li, result):
+    from doubly_linked import DoublyLinked
+    new_list = DoublyLinked()
+    new_list.insert(li)
+    assert new_list.size() == 4
```

Comment:
What other tests could you write to prove `.insert()` is working? This shows what the size is, but are we sure our actual nodes are actually in there? Prove it with more testing.